src/main/java/com/dnsimple/endpoints/Domains.java

```diff
@@ -107,7 +109,9 @@ public PaginatedResponse<Collaborator> listCollaborators(Number account, String
      * @param options The options for the list request
      * @return The list collaborators response
      * @see <a href="https://developer.dnsimple.com/v2/domains/collaborators/#listCollaborators">https://developer.dnsimple.com/v2/domains/collaborators/#listCollaborators</a>
+     * @deprecated `DomainCollaborators` have been deprecated and will be removed in the next major version. Please use our Domain Access Control feature.
      */
+    @Deprecated(since = "2.0.0", forRemoval = true)
     public PaginatedResponse<Collaborator> listCollaborators(Number account, String domain, ListOptions options) {
```

Comment:
This seems to be a duplicate (we don't have an endpoint for account collaborators.)